### PR TITLE
bpo-39710: calendar documentation

### PR DESCRIPTION
**Pull request by dorosch**
https://bugs.python.org/issue39710

### CODE REVIEW

File: Misc/NEWS.d/next/Documentation/2020-02-23-16-40-51.bpo-39710.GU_1Bb.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+"will be returned as unicode" reminiscent from Python 2
```
Comment (epicfaace) [inline Misc/NEWS.d/next/Documentation/2020-02-23-16-40-51.bpo-39710.GU_1Bb.rst]:
```suggestion
Removed from the calendar documentation "will be returned as unicode" wording reminiscent of Python 2
```

**Review by epicfaace — APPROVED** [review]

**Comment by csabella** [comment]
@dorosch, please make the same change to the documentation files in the Docs folder.  Thank you!

File: Misc/NEWS.d/next/Documentation/2020-02-23-16-40-51.bpo-39710.GU_1Bb.rst (hunk 1 above)
Comment (iritkatriel) [reply Misc/NEWS.d/next/Documentation/2020-02-23-16-40-51.bpo-39710.GU_1Bb.rst]: I think we can skip news on this change.

**Comment by JulienPalard** [comment]
@dorosch Do you have the time to look at the previous comment from@csabella? It would be great.

**Comment by ambv** [comment]
Closing in favor of GH-26985.

**Pull request closed without merging**